shards/modules/gfx/shader/core_shards.hpp

```diff
@@ -565,6 +608,34 @@ template <typename TShard> struct Write : public IOBase {
   }
 };
 
+struct ShaderWriteOutput: public Write<blocks::WriteOutput> {
+  static SHOptionalString help() {
+    return SHCCSTR("This shard sets the value passed as input to the vertex attribute specified in the Name parameter and outputs it to the next stage or render target.");
```

Comment:
In the fragment stage will write to one of the outputs attached to the render pass.
In the vertex stage will write or create a new output that can later be read from the fragment stage.

Above is more accurate. The shard doesn't set any vertex attributes